doc/cephadm/install.rst

```diff
@@ -121,6 +121,13 @@ The default bootstrap behavior will work for the vast majority of
 users.  See below for a few options that may be useful for some users,
 or run ``cephadm bootstrap -h`` to see all available options:
 
+* In production environments it is recommended to have network separation
+  between the public network traffic and the cluster traffic which handles
```

Comment:
I'm not sure I agree with the `it is recommended` part.  IMO we should push users to use a single network for simplicity.. maybe with bonding.

Reply:
single network would be simpler - no argument there. However, we also state "significant performance improvement with a second "cluster" network in a large cluster" in the network configuration docs. Separation also lends itself to fault isolation and impact, QoS etc etc
I'll reword this text thought to align better with the network guide statement.

Reply:
Right.  TBH I thought we had changed that language to not suggest separate networks. @markhpc @mmgaggle what do you think?